docs/part3-nextflow.md

```diff
@@ -0,0 +1,74 @@
+# Part 3. Critter classification with Nextflow
+
+## Introduction to Running a Nextflow Workflow
+
+Nextflow is a workflow tool that enables the orchestration of complex computational pipelines with ease. It provides an abstraction layer over the underlying execution environment, whether it's your local computer, a high-performance computing cluster, or cloud infrastructure. This means you can write your workflow once and run it anywhere, ensuring reproducibility across different environments.
+
+<!-- Talk about more nf things here -->
```

Comment:
```suggestion
Importantly this approach addresses a number of the problems you might have noticed using the Bash scripting:

 * Processes will run in parallel, limited only by available compute resources (it takes some effort to make that happen in Bash). In fact you often use compute resources outside of the machine you execute the workflow on.
 * Software is automatically made available specific to each process. You might not have appreciated that in the previous examples because we prepared your software environment for you, but it's a very important consideration.
```